packages/sdk/src/swap/types.ts

```diff
@@ -35,9 +35,7 @@ export interface QuoteResponse extends Route {
   quote: QuoteQueryResponse;
 }
 
-export interface SwapRequest extends Omit<QuoteResponse, 'quote'> {
-  expectedDepositAmount: string;
-}
+export type SwapRequest = QuoteRequest;
```

Comment:
i think it would make sense to move the `destAddress` to the `SwapRequest` type

Reply:
we already "make up" a destination address in the swap app because of this. i think it will be the same for others that are trying to use it